recipes/xnvme-sys/meta.yaml

```diff
@@ -0,0 +1,74 @@
+---
+{% set name = "xnvme" %}
+{% set version = "0.7.4" %}
+{% set sha256 = "6d42d0bd7e6b395a37869a8713d138d332500f20ecc272ebf6789471026a7191" %}
+
+package:
+  name: {{ name|lower }}-sys
```

Comment:
What's the reason in adding `sys` here?

Reply:
The intent was to provide this package for the system-libraries and tools that **xNVMe** provides and then a package named **xnvme** for the Python bindings. However, it probably makes better sense for this package to be **xnvme** and then the other package can be **xnvme-py**, or include it in **xnvme** package? What is the convention to follow here?

For now, then i'll rename it to **xnvme** instead of **xnvme-sys**.

Reply:
That makes sense. Name it libxvme instead. That is our usual convention for these kind of packages.

Reply:
ack.